enterprise-suite/templates/es-console.yaml

```diff
@@ -39,8 +40,29 @@ spec:
   selector:
     run: es-console
   type: NodePort
-{{ end }}
 
+{{ else }}
```

Comment:
why is this conditional?  what does the complexity cost of another option buy us in this case?  Is there some concern that there is a typical case where we wouldn't want to define this service, or we'd be unpleasantly surprised if this were defined?

I'd just define the service, for all cases, no condition.